src/lib/battery/module.yaml

```diff
@@ -129,12 +129,13 @@ parameters:
                     This parameter controls the source of battery data. The value 'Power Module'
                     means that measurements are expected to come from a power module. If the value is set to
                     'External' then the system expects to receive mavlink battery status messages.
+                    If the value is set to 'ESCs', the battery information are taken from the esc_status message.
```

Comment:
Might be worth mentioning that this includes both voltage and current (and therefore the ESC must provide both).